lua/entities/gmod_wire_expression2/core/custom/cfc_entity.lua

```diff
@@ -40,6 +40,22 @@ e2function number entity:isInFaction()
     return this:IsInFaction()
 end
 
+local whitelist = {
+    ["prop_physics"] = true
+}
+e2function entity:propIgnite(duration)
+    duration = duration or 30
```

Comment:
I don't think this is necessary, because of how e2 functions work duration should always be a number.

Reply:
I think you need to make a new function without a parameter instead

Reply:
I was thinking this would create a default duration, so you could run propIgnite without passing a param and still work fine using a default of 30.

Reply:
this would work for a lua function but with e2 parameters arent optional.
 you would need to define 2 functions `e:propIgnite(n)` and `e:propIgnite()`

Reply:
Ah okay I see now. So what would you guys prefer, adding another function without a param, only allowing people to ignite for a duration of 30, or just remove line 47?

Reply:
I think two functions is the way to go